src/Akka.Persistence.Sql/Db/AkkaPersistenceDataConnectionFactory.cs

```diff
@@ -169,22 +180,6 @@ public AkkaPersistenceDataConnectionFactory(IProviderConfig<SnapshotTableConfigu
                     .HasLength(100);
             }
 
-            if (config.ProviderName.ToLower().Contains("sqlite"))
-            {
-                journalRowBuilder
-                    .Member(r => r.Ordering)
-                    .IsPrimaryKey()
-                    .HasDbType("INTEGER")
-                    .IsIdentity();
-            }
-            else
-            {
-                journalRowBuilder
```

Comment:
We still need to have a unique index for PersistenceId / SeqNo - necessary for EventsByPersistenceId and recoveries. Is that covered somewhere?

Reply:
Yes, the unique index is appended as an extra SQL command in the footer